plenum/server/replica.py

```diff
@@ -2786,6 +2855,8 @@ def revert_unordered_batches(self):
                 discarded = invalid_index_serializer.deserialize(discarded)
                 self.logger.debug('{} reverting 3PC key {}'.format(self, key))
                 self.revert(ledger_id, prevStateRoot, len_reqIdr - len(discarded))
+                pp = self.getPrePrepare(*key)
+                self._consensus_data_helper.clear_batch(pp)
```

Comment:
Not sure that we should call it here....

Reply:
ok, removed